Week3/homework/step4-bonus.js

```diff
@@ -3,8 +3,14 @@
 const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 
 // Add your function here. Try and come up with a good name for this function
-
+function removeDoubles(foo) {
+    const uniqueLetters = [];
+    for (let i = 0; i < foo.length; i++){
+        uniqueLetters.push(foo[i]);
```

Comment:
All you are doing here is copying the whole array. *Not* removing the duplicate values.
You need to check first whether the value already exists in `uniqueLetters` and if it does just skip to the next iteration of the loop, and if it does not, add it to `uniqueLetters`.